torch/csrc/distributed/c10d/init.cpp

```diff
@@ -2784,6 +2785,7 @@ Example::
     >>> nccl_options.config.cga_cluster_size = 2
     >>> nccl_options.config.max_ctas = 4
     >>> nccl_options.config.min_ctas = 2
+    >>> nccl_options.config.split_share = 1
```

Comment:
I would put `split_share = 0` for the example here since you plan to put the default to `1`.

Reply:
In this version of code, split_share = 0 is the default